hexane/force_matching/run.sh

```diff
@@ -0,0 +1,37 @@
+#! /bin/bash -e
```

Comment:
For all other tutorials the `fmatch.xml` and force matching example is in `atomistic` and so I would keep it like this for hexane due to consistency.